packages/match-media/src/useResponsiveValue.js

```diff
@@ -0,0 +1,11 @@
+import { useThemeUI } from 'theme-ui'
+import useBreakpointIndex from './useBreakpointIndex'
+
+const useResponsiveValue = values => {
```

Comment:
Thanks for working on this! Why not keep all the code in a single `src/index.js` file?

Reply:
No real reason I guess, I usually like to write one hook per file. Happy to do that if you prefer.

Reply:
Mostly just curious here – personally, I think it makes sense to keep these together until there is a good reason to split into multiple files, but this is fine as-is

Reply:
I figured I might as well update it, in this case I also prefer them in a single file